.github/actions/multi-approvers/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "multi-approvers",
+  "version": "1.0.0",
+  "description": "GitHub Action that enforces approver requriements.",
+  "author": "",
+  "license": "Apache-2.0",
+  "main": "src/index.js",
+  "scripts": {
+    "tests": "jest",
+    "build-min": "ncc build -o dist/main -m src/index.js",
+    "build": "ncc build -o dist/main src/index.js"
```

Comment:
If we're going to go to the trouble of compiling, we should also just use Typescript for type safety too. Then we can use eslint, etc.

Reply:
Done.